atoma-event-subscribe/sui/src/subscriber.rs

```diff
@@ -228,12 +238,7 @@ impl SuiSubscriber {
             .send((request.params().prompt(), oneshot_sender))
             .await
             .map_err(Box::new)?;
-        let result = tokio::time::timeout(
-            Duration::from_secs(WAIT_FOR_INPUT_MANAGER_RESPONSE_SECS),
-            oneshot_receiver,
-        )
-        .await
-        .map_err(|_| SuiSubscriberError::TimeoutError)??;
+        let result = oneshot_receiver.await??;
```

Comment:
Why is this better ?

Reply:
because when the `AtomaInputManager` timeouts it will send the timeout error. So I don't have to have timeout here. Before it just didn't send anything so I had to have the timeout here as well.

Reply:
But doesn't this way also panics the node service ? You are still crashing on error, no ?

Reply:
No, the error is handled properly on line 201

Reply:
Agree !